lib/libvcc/vcc_expr.c

```diff
@@ -146,13 +146,10 @@ vcc_DoubleVal(struct vcc *tl)
 void
 vcc_Duration(struct vcc *tl, double *d)
 {
-	double v, sc;
```

Comment:
Not sure I follow. Mind sharing some VCL where this is visible?
All examples I can think of require dropping the unit, and in any case we should be consistent.